src/org/usfirst/frc/team166/robot/subsystems/Drive.java

```diff
@@ -87,36 +87,40 @@ public void turnAngle(double desiredAngle) {
 
 	public void turnAngleCCW() {
 		angleError = Math.abs(angleError);
-
-		if (angleError >= 20.0) {
-			setMotorPower(1, -1, 1, -1);
-
-		} else if ((angleError <= 20.0 && angleError >= 5.0) || (angleError >= -20.0 && angleError <= -5.0)) {
-			setMotorPower(angleError / 20.0, -angleError / 20.0, angleError / 20.0, -angleError / 20.0);
-
-		} else if ((angleError < 5.0 && angleError >= 0.1) || (angleError > -5.0 && angleError <= -0.1)) {
-			setMotorPower(0.25, -0.25, 0.25, -0.25);
-
-		} else if (angleError < 0.1 && angleError > -0.1) {
-			stopMotors();
-		}
+		//
+		// if (angleError >= 20.0) {
+		// setMotorPower(1, -1, 1, -1);
+		//
+		// } else if ((angleError <= 20.0 && angleError >= 5.0) || (angleError >= -20.0 && angleError <= -5.0)) {
+		// setMotorPower(angleError / 20.0, -angleError / 20.0, angleError / 20.0, -angleError / 20.0);
+		//
+		// } else if ((angleError < 5.0 && angleError >= 0.1) || (angleError > -5.0 && angleError <= -0.1)) {
+		// setMotorPower(0.25, -0.25, 0.25, -0.25);
+		//
+		// } else if (angleError < 0.1 && angleError > -0.1) {
+		// stopMotors();
+		// }
+		setMotorPower(motorCompAngle(angleError), -motorCompAngle(angleError), motorCompAngle(angleError),
+				-motorCompAngle(angleError));
 	}
 
 	public void turnAngleCW() {
 		angleError = -Math.abs(angleError);
 
-		if (angleError >= 20.0) {
-			setMotorPower(1, -1, 1, -1);
-
-		} else if ((angleError <= 20.0 && angleError >= 5.0) || (angleError >= -20.0 && angleError <= -5.0)) {
-			setMotorPower(angleError / 20.0, -angleError / 20.0, angleError / 20.0, -angleError / 20.0);
-
-		} else if ((angleError < 5.0 && angleError >= 0.1) || (angleError > -5.0 && angleError <= -0.1)) {
-			setMotorPower(0.25, -0.25, 0.25, -0.25);
-
-		} else if (angleError < 0.1 && angleError > -0.1) {
-			stopMotors();
-		}
+		// if (angleError >= 20.0) {
```

Comment:
Don't leave commented-out code in - we can go back in time to get it if we need it